install/homebrew.md

```diff
@@ -73,3 +80,21 @@ Dserver.context-path=/
 Dairsonic.home=#{workingdir}
 Djava.awt.headless=true
 ```
+
+##### Updating Runtime settings
+
+Use your favorite text editor and open _$(brew --prefix)/Cellar/airsonic/{{ site.stable_version }}/homebrew.mxcl.airsonic.plist_
```

Comment:
Use back quotes to specify a file name please.

Reply:
sure thing, i'll update it all now. thanks!